src/_includes/config/setup-cron_how-to.md

```diff
@@ -31,7 +37,7 @@ A sample follows:
 
 ```terminal
 #~ MAGENTO START c5f9e5ed71cceaabc4d4fd9b3e827a2b
-* * * * * /usr/bin/php /var/www/html/magento2/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> /var/www/html/magento2/var/log/magento.cron.log
+* * * * * /usr/bin/php<php_version> <install_directory>/bin/magento cron:run 2>&1 | grep -v "Ran jobs by schedule" >> <install_directory>/var/log/magento.cron.log
```

Comment:
Although generalizing would be nice, the full path seems to be used heavily throughout the cron topics. You'd have to address the consistency, so maybe this change is out of scope.

Reply:
Typically, we use generalization to describe a format of a command or something else. When we are talking about a sample, we provide a particular case without generalization.